pkg/routes/logs.go

```diff
@@ -43,7 +43,7 @@ func (l Logs) Install(c *restful.Container) {
 
 func logFileHandler(req *restful.Request, resp *restful.Response) {
 	logdir := "/var/log"
-	actual := path.Join(logdir, req.PathParameter("logpath"))
+	actual := filepath.Join(logdir, req.PathParameter("logpath"))
```

Comment:
does this do the right thing on windows when joining a base path of `/var/log`?

Reply:
a couple more questions:
* does http.ServeFile handle backslashed paths correctly?
* what does filepath.Join do when logdir and req.PathParameter("logpath") contain a mix of forward-slash and backslash separators?

Reply:
@claudiubelu please address this

Reply:
Looking through ``http.ServeFile``'s code, it seems that there are some explicit checks for `/`, including for `/index.html`. Removed this change.